src/main/java/bio/terra/service/snapshotbuilder/utils/constants/ConceptAncestorConstants.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.service.snapshotbuilder.utils.constants;
+
+public final class ConceptAncestorConstants {
+  private ConceptAncestorConstants() {}
+
+  public static final String CONCEPT_ANCESTOR = "concept_ancestor";
```

Comment:
Since this field is the name of the table, and this class is the table, I would call this field `NAME`
```suggestion
  public static final String NAME = "concept_ancestor";
```

Reply:
I don't have a strong opinion on this!

Reply:
Seconding. Or `TABLE_NAME`